### PR TITLE
SVCPLAN-1230 Add telegraf gridftp_log_parse.sh script

### PR DESCRIPTION
**Pull request by bsper2**
- Add telegraf gridftp_log_parse.sh script
- Add CI/CD
- Minor yamllint fixes
- Update README.md
- Add REFERENCE.md

### CODE REVIEW

**Comment by bsper2** [comment]
This has been tested on dt-dtn[1-4]

File: templates/gridftp_log_parse_config.epp [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%- | $log_file_path,
+      $endpoint,
+| -%>
+# This file is managed by Puppet
+
+##Globus Telegraf Config
```
Comment (jakerundall) [inline templates/gridftp_log_parse_config.epp]: "##Globus Telegraf Config" should probably be changed to something more specific as this config file & script have a more specific purpose.

File: templates/gridftp_log_parse_config.epp [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+##
+## Example [ Description]
```
Comment (jakerundall) [inline templates/gridftp_log_parse_config.epp]: Might as well add a space between "Description" and closing "]".

**Review by jakerundall — APPROVED** [review]
Looks good, just a few "nits" I commented on.

File: templates/gridftp_log_parse_config.epp (hunk 1 above)
Comment (bsper2) [reply templates/gridftp_log_parse_config.epp]: True, updated it to `Globus Log Parse Telegraf Config`

**Pull request merged**